src/test/scala-2.12/org/apache/james/gatling/simulation/smtp/SmtpNoAuthenticationNoEncryptionBigBodySimulation.scala

```diff
@@ -4,22 +4,22 @@ import io.gatling.core.Predef._
 import org.apache.james.gatling.control.{UserCreator, UserFeeder}
 import org.apache.james.gatling.simulation.Configuration
 import org.apache.james.gatling.smtp.SmtpProtocol.smtp
-import org.apache.james.gatling.smtp.scenari.NoAuthenticationNoEncryptionScenario
+import org.apache.james.gatling.smtp.scenari.SmtpNoAuthenticationNoEncryptionScenario
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration.Duration.Inf
 import scala.concurrent.duration._
 import scala.concurrent.{Await, Future}
 
-class NoAuthenticationNoEncryptionBigBodySimulation extends Simulation {
+class SmtpNoAuthenticationNoEncryptionBigBodySimulation extends Simulation {
 
   private val users = Await.result(
     awaitable = Future.sequence(
       new UserCreator(Configuration.BaseJamesWebAdministrationUrl).createUsersWithInboxAndOutbox(Configuration.UserCount)),
```

Comment:
Idem as for IMAP

You don't want to create users, you just want to use the existing one already created in the provisioned data.
You can use the users in SimulationOnMailCorpus like it's done on the JMAP simulations